schemas/json/aas.json

```diff
@@ -598,7 +598,7 @@
         },
         {
           "properties": {
-            "annotation": {
+            "annotations": {
```

Comment:
There are no annotations, they are called "**extensions**" (annotation was an alternative naming discussed in the beginning)